### DISCUSSION TITLE
`:CodeCompanion` output is inlcuding the backtick markdown code fences

### DISCUSSION THREAD

**Discussion by mrjones2014** [olimorris/codecompanion.nvim#14] [Q&A] [answered]
Is this something wrong with my config, or do we need to update the system prompt to tell the model not to include the triple-backticks?

**Comment by olimorris** [comment]
I think the backticks may need to be explicitly called out. The prompt does ask for no codeblocks but some models have been super resistant to that. GPT-4 went through a phase of completely ignoring it before reverting back to normal.

The `init.lua` system prompt, line 27, is the one to adjust. Can you see what works for you and then we can include it.

I _really_ don't want to get to having different prompts for different models.

**Reply by mrjones2014** [reply]
Gotcha, it seems like `llama2` is just very resistant to it. I would probably recommend `dolphin-mixtral` as the default model for code, it's much better than `llama2`. Part of its system prompt is to follow instructions absolutely no matter what, its #1 priority is following the users instructions, which helps with this kind of thing.

**Reply by olimorris** [reply, marked as answer]
Good shout. I've now made that the default.
